sample_data/project-list.md

```diff
@@ -166,10 +170,12 @@ Wikimedia sample data (copy from list above):
 
 - [OHLOOM](https://wiki.opensourceecology.de/Open_Hardware-Webstuhl_%E2%80%93_OHLOOM)
 
-### tracing & tracking
+### tracing & tracking & remote sensing
 
 - [AudioMoth](https://www.openacousticdevices.info/audiomoth)
 - [bGeigieNanoKit](https://github.com/Safecast/bGeigieNanoKit)
+- [SenseBox](https://sensebox.de/)
```

Comment:
Already included under `lab equipment`, but it makes much more sense here.
As this list is growing we may want to restructure it anyway :)

I propose the IPC system (International Patent Classification)
- relatively stable & internationally recognised
- applicable for all kinds of hardware
- data field in the requested metadata that certainly no one will provide 'out of stock', so people may be happy if we make a proposition.

However, that's another issue :)

Reply:
OK. I've removed SenseBox from lab equipment for now. But yeah some re-structuring would be nice. :)